deepchecks/tabular/checks/train_test_validation/multivariate_drift.py

```diff
@@ -12,14 +12,14 @@
 import warnings
 
 from deepchecks.core import CheckResult, ConditionCategory, ConditionResult
-from deepchecks.core.check_utils.whole_dataset_drift_utils import run_whole_dataset_drift
+from deepchecks.core.check_utils.multivariate_drift_utils import run_multivariate_drift
 from deepchecks.tabular import Context, TrainTestCheck
 from deepchecks.utils.strings import format_number
 
-__all__ = ['WholeDatasetDrift']
+__all__ = ['MultivariateDrift']
 
 
-class WholeDatasetDrift(TrainTestCheck):
+class MultivariateDrift(TrainTestCheck):
```

Comment:
Would add a deprecation warning to the old name and reference to new check name, can see an example for that in: deepchecks/tabular/checks/model_evaluation/performance_report.py

Reply:
Should I add the warning message on the `__init__` method of `MultivariateDrift` class or similar to `PerformanceReport` from deepchecks/tabular/checks/model_evaluation/performance_report.py ?

Reply:
@Nadav-Barak I have made the requested change, could you check if that's fine ?